app/Presenters/templates/@layout.latte

```diff
@@ -26,10 +26,48 @@
 <body>
 <script> document.documentElement.className += ' js'</script>
 <div class="container">
-    <h1>Application layout</h1>
+    <nav class="navbar navbar-default navbar-static-top">
+        <div class="container">
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                <a class="navbar-brand" href="#">Instante</a>
+            </div>
+            <div id="navbar" class="navbar-collapse collapse">
+                <ul class="nav navbar-nav">
+                    <li class="active"><a href="#">Home</a></li>
+                    <li class="dropdown">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dropdown <span class="caret"></span></a>
+                        <ul class="dropdown-menu">
+                            <li><a href="#">Action</a></li>
+                            <li><a href="#">Another action</a></li>
+                            <li><a href="#">Something else here</a></li>
+                            <li role="separator" class="divider"></li>
+                            <li class="dropdown-header">Nav header</li>
+                            <li><a href="#">Separated link</a></li>
+                            <li><a href="#">One more separated link</a></li>
+                        </ul>
+                    </li>
+                </ul>
+
+            </div>
+        </div>
+    </nav>
+
+
+        <div class="jumbotron">
+            <h1> Welcome to Instante</h1>
+            <p>Instante skeleton is ready, you can start creating your application now.
+        </div>
+
     {include #content}
 </div>
 {control jsLoader}
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```

Comment:
please do not use cdn bootstrap:

- inject JsModuleContainer to BasePresenter and `$this->jsModuleContainer->useModule('bootstrap3');` to BasePresenter::beforeRender
- add reference to bootstrap 3 to require-config.js: `bootstrap3: '../../bower_components/bootstrap/dist/js/bootstrap.min',`
- compile frontend using "gulp" command inside frontend folder